heartbeat/docs/monitors/monitor-browser.asciidoc

```diff
@@ -0,0 +1,105 @@
+[[monitor-browser-options]]
+=== Browser options
+
+Also see <<monitor-options>>.
```

Comment:
@bmorelli25 can you cross link to https://github.com/elastic/observability-docs/pull/609 ? These reference docs are great, but really, people can't use this without the quickstart.

Reply:
👍 Added https://github.com/elastic/beats/pull/25506/files#r626119889 with a suggestion to move the link to `<<monitor-options>>` down on the page and instead link to the quick start guide up top.